packages/server/src/index.ts

```diff
@@ -20,11 +17,6 @@ export async function main(configName: string): Promise<void> {
 
   const config = await loadConfig(configName);
 
-  // Preload the schema
```

Comment:
Oy, we were double loading the schema? 🤦‍♂️   Nice catch.

IIRC, we did intentionally load the schema before initializing `express`, to the case where we lazy load the schema inside an HTTP request.  Maybe we were doing that anyway (sigh 🤦‍♂️), but I think the theory still holds?  So might be worth adding `getStructureDefinitions()` here?